.gitignore

```diff
@@ -21,3 +21,4 @@ apk-sizes-*.txt
 *.binlog
 src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props
 *~
+monodroid/
```

Comment:
This should be `external/monodroid`, otherwise I fear it'll try to ignore `src/monodroid`, which would be Bad™.